wrappers/python/zxing.cpp

```diff
@@ -83,11 +119,11 @@ PYBIND11_MODULE(zxing, m)
 		.value("DataBarExpanded", BarcodeFormat::DataBarExpanded)
 		.value("UPCA", BarcodeFormat::UPCA)
 		.value("UPCE", BarcodeFormat::UPCE)
-		// use upper case 'NONE' because 'None' is a reserved identifyer in python
+		// use upper case 'NONE' because 'None' is a reserved identifier in python
 		.value("NONE", BarcodeFormat::None)
 		.value("OneDCodes", BarcodeFormat::OneDCodes)
 		.value("TwoDCodes", BarcodeFormat::TwoDCodes)
-		.export_values()
+		//.export_values()
```

Comment:
The export_values() do export the enumeration values outside the enum (i.e. at python module level).
This make all theses values accessible either using zxing.BarcodeFormat.OneDCodes or zxing.OneDCodes.
As there are several enumerations, I find it cleaner that each enumeration value is present only on enumeration itself and are not mixed together at the module level.
This is my opinion and can be changed if you consider it better to have enumeration values at the module level.

Reply:
Ah, now I remember. I don't think this would harm the cleanness of the API as it is very small. On the other hand, I did forget about it and it was not used in the `demo_writer.py` nor in the `test.py`. But there is also the backward compatibility argument to be made... Please leave it as it was before.

Reply:
Done